modules/node_modules/@ncigdc/containers/CancerDistributionTable.js

```diff
@@ -1,32 +1,72 @@
 // @flow
 import React from 'react';
 import Relay from 'react-relay';
+import { lifecycle, compose } from 'recompose';
+import { groupBy, isEqual } from 'lodash';
 import { buildCancerDistributionData } from '@ncigdc/utils/data';
 import { tableToolTipHint } from '@ncigdc/theme/mixins';
 import EntityPageHorizontalTable from '@ncigdc/components/EntityPageHorizontalTable';
 import { Tooltip } from '@ncigdc/uikit/Tooltip';
 import { Column } from '@ncigdc/uikit/Flex';
 import { makeFilter } from '@ncigdc/utils/filters';
+import ProjectLink from '@ncigdc/components/Links/ProjectLink';
+import MutationsCount from '@ncigdc/containers/MutationsCount';
 
 type TProps = {|
   node: Object,
   aggregations: Object,
 |};
 
-const CancerDistributionTableComponent = ({
+const CancerDistributionTableComponent = compose(
+  lifecycle({
+    componentDidMount(): void {
+      this.props.relay.setVariables({
```

Comment:
I don't think so http://stackoverflow.com/questions/34255662/props-in-relay-preparevariables